scripts/travis-ci/run-test.sh

```diff
@@ -13,6 +13,9 @@ case "$1" in
     *)
         ln -s $MODULE_DIR $DRUPAL_DIR/modules/og
         cd $DRUPAL_DIR
+        git clone --branch 8.x-1.x https://git.drupal.org/project/composer_manager.git modules/composer_manager
```

Comment:
I'll try construct something with composer manager but I have a feeling this will make things much more complex.